main.rb

```diff
@@ -0,0 +1,77 @@
+require_relative 'book_app'
+require_relative 'music_album_app'
+require_relative 'game_app'
+
+class Main
+  def initialize
+    @book_app = BookApp.new
+    @music_app = MusicApp.new
+    @game_app = GameApp.new
+  end
+
+  def display_menu
+    puts 'Options:'
+    puts '1. Create new book'
+    puts '2. Create new music album'
+    puts '3. Create new movie'
+    puts '4. Create new game'
+    puts '5. List all books'
+    puts '6. List all music albums'
+    puts '7. List all movies'
+    puts '8. List of games'
+    puts '9. List all genres'
+    puts '10. List all labels'
+    puts '11. List all authors'
+    puts '12. List all sources '
+    puts '13. Quit'
+  end
```

Comment:
- [ ] If I delete the local storage files and start the app I get the below error. Please make sure to handle the situation when one or more storage files are missing, the app should work as expected. You're handling this for `books` and `labels` cases but for others, there's an error.

![image](https://github.com/gaks1/ruby-group-capstone/assets/83803414/37d786a8-9978-444a-baec-6f46f81b0648)